jl4/examples/legal/tests/britishcitizen2.golden

```diff
@@ -1,7 +1,7 @@
 Parsing successful
 Typechecking successful
 Evaluation successful
-britishcitizen2.l4:54:7-17:
+ britishcitizen2.l4:54:7-17:
```

Comment:
This looks wrong. We should check where this space is coming from, but it certainly shouldn't be there and needlessly change all the test results.

Reply:
Yeah that was lazy, you’re totally right